VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_:


BO_ 336 HEARTBEAT: 8 Vector__XXX
 SG_ State : 0|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ Counter : 8|8@1+ (1,0) [0|0] "" Vector__XXX

BO_ 337 ERRORS: 8 Vector__XXX
 SG_ BSPD_Error : 0|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ LowPressure_Error : 8|8@1+ (1,0) [0|0] "" Vector__XXX

BO_ 338 DIGITAL: 8 Vector__XXX
 SG_ HighPressure : 0|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ LowPressure : 8|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ FiveKW : 16|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ BSPD_OK : 24|8@1+ (1,0) [0|0] "" Vector__XXX

BO_ 340 ANALOGUE: 8 Vector__XXX
 SG_ Brake1_Percent : 0|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ Brake2_Percent : 8|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ Brake_Average : 16|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ Brake1_Voltage : 24|8@1+ (0.1,0) [0|0] "" Vector__XXX
 SG_ Brake2_Voltage : 32|8@1+ (0.1,0) [0|0] "" Vector__XXX




